domains/certificates/Query.API/API.IntegrationTests/Factories/QueryApiWebApplicationFactory.cs

```diff
@@ -110,13 +111,13 @@ public HttpClient CreateAuthenticatedClient(string subject)
         return tokenHandler.WriteToken(token);
     }
 
-    public async Task AddContract(string subject, string gsrn, DateTimeOffset startDate,
+    public async Task AddContract(string subject, string gsrn, DateTimeOffset startDate, MeteringPointType meteringPointType,
         DataSyncWireMock dataSyncWireMock)
     {
-        dataSyncWireMock.SetupMeteringPointsResponse(gsrn: gsrn);
+        dataSyncWireMock.SetupMeteringPointsResponse(gsrn: gsrn, type: meteringPointType.ToString());
 
         using var client = CreateAuthenticatedClient(subject);
-        var body = new { gsrn, startDate = startDate.ToUnixTimeSeconds() };
+        var body = new { gsrn, startDate = startDate.ToUnixTimeSeconds(), meteringPointType = meteringPointType.ToString() };
```

Comment:
Is the meteringPointType needed here?

Reply:
Yes - need it for CertificateIssuinTests where I need to create contracts for consumption

Reply:
As far as I can see, this field in the body is not used in the ContractsController.

Reply:
True - not needed on this particular line. Corrected